README.md

```diff
@@ -34,20 +34,20 @@ The easiest way to manage multiple node versions in development is to use
 ## Get started
 
 * type `yarn` to install all dependencies
-* type `yarn dev:amo:dev` to start a local dev server that connects to a
-  hosted development environment
+* type `yarn amo:stage` to start a local server that connects to a
```

Comment:
I think stage data is probably the best for newbies since it's cleaner and has icons (etc.)